kalite/distributed/static/js/distributed/intro-injector.js

```diff
@@ -0,0 +1,122 @@
+//on page load
+$(function() {
+  console.log( "ready!" );
+
+  //Use relative URL to determine which YAML file to parse
+  var relative_url = window.location.pathname;
+  var response = getIntro(relative_url);
+  
+  //Parse YAML file into JSON to be used with intro.js !!! this is actually just a json object for nao.
+  var jsonResponse = parseResponse(response);
+
+console.log(jsonResponse);
+
+  //Load from our new JSON object, and attach to elements in page
+  attachAttributes(jsonResponse);
+
+  console.log("uhh hello?");
+
+});
+
+function startTour() {
```

Comment:
You should add explanatory comments about functions. We don't seem to have a set-in-stone style for JS yet, but at a minimum I would say indicate what the expected parameters are (in this case none), what the intended use of the function is and its side effects, and what the function returns.